packages/builtin/array.pony

```diff
@@ -1,6 +1,36 @@
 class Array[A] is Seq[A]
   """
   Contiguous, resizable memory to store elements of type A.
+
+  # Usage
+  Creating an array of Strings, single line.
+  ```pony
+    let array: Array[String] = ["one"; "two"; "three"]
+  ```
+
+  Semicolon terminates the line, so this is a mulitiline equivalent.
```

Comment:
Note: semicolon's are expression separators not terminators.

Reply:
Thanks! Fixing it.